### PR TITLE
add arch MAN and its yml

### PR DESCRIPTION
**Pull request by TIANLE233**

### CODE REVIEW

File: archs/man_arch.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+        res = x.clone()
+
+        for i, block in enumerate(self.body):
+            res = block(res)
+        x = self.body_t(res) + x
+
+        return x
+
+
+# @ARCH_REGISTRY.register()
```
Comment (liozur) [inline archs/man_arch.py]: Fix me.

File: options/expe/MAN/MAN-light_LSR.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+name: MAN-light
+model_type: IRModel
+num_gpu: 1  # 0 for cpu mode
+manual_seed: 42
+bit: 8 # data type
+
+
+# network structure ####################################################################################################
+network_g:
+  type: MAN-light
```
Comment (liozur) [inline options/expe/MAN/MAN-light_LSR.yml]: Fix me.

File: archs/man_arch.py (hunk 1 above)
Comment (TIANLE233) [reply archs/man_arch.py]: I may have fixed it, please check

File: options/expe/MAN/MAN-light_LSR.yml (hunk 2 above)
Comment (TIANLE233) [reply options/expe/MAN/MAN-light_LSR.yml]: I may have fixed it, please check

**Comment by liozur** [comment]
please add the bibtex of MAN

**Comment by liozur** [comment]
Thx.

**Pull request merged**